packages/graphql-mocks/src/graphql/handler.ts

```diff
@@ -78,16 +85,19 @@ export class GraphQLHandler {
       schema,
       variableValues,
       contextValue,
+      fieldResolver: fieldResolverRouter,
+      typeResolver: typeResolverRouter,
     }) as ExecutionResult<DataResult>;
   }
 
-  protected async pack(): Promise<void> {
+  async pack(): Promise<void> {
```

Comment:
By making this public it'll allow for ahead of time packing of needed resources instead of deferring to the first query.